index.js

```diff
@@ -141,14 +145,19 @@ function serveApiDocs() {
   spec.definitions = mongooseModelsSpecs || {};
   updateSpecFromPackage();
   spec = patchSpec(predefinedSpec);
-  app.use(packageInfo.baseUrlPath + '/api-spec', (req, res, next) => {
```

Comment:
Is it possible to have next line like `specV3 = convertOpenApiVersionToV3(spec, specV3);` and then serve it separately as described in the comment in the issue?

Reply:
Done!